### PR TITLE
1648 travis fail build

### PR DESCRIPTION
**Pull request by ViktarR**

### CODE REVIEW

**Comment by mf-code** [comment]
[5434] - kyeTIECL

**Comment by mf-code** [comment]
[5434] - Allure report on Netlify: https://5ece852b83e8e9a599122974--musing-swartz-6fc060.netlify.app

**Comment by mf-code** [comment]
[5435] - OCY3VyHI

**Comment by mf-code** [comment]
[5435] - Allure report on Netlify: https://5ece8a23d562c6079c3d78bd--musing-swartz-6fc060.netlify.app

File: .travis.yml [hunk 1]
```diff
@@ -49,19 +49,19 @@ jobs:
...
-        - echo arguments=$CHROME_ARGS > "./$BDD_TESTS/src/test/resources/$CHROME_PROPERTIES"
-        - mvn verify $WITH_PARAMS -pl $GROUP_ID:$BDD_TESTS
-
-    - stage: test
-      name: Run tests - bootstrap
-      script:
-        - mvn -ntp install -DskipTests
-        - echo arguments=$CHROME_ARGS > "./$BOOTSTRAP_TESTS/src/test/resources/$CHROME_PROPERTIES"
-        - mvn verify $WITH_PARAMS -pl $GROUP_ID:$BOOTSTRAP_TESTS
+#    - stage: test
```
Comment (AkioFaris) [inline .travis.yml:52]: Uncomment tests stages that were commented for debug purposes

File: reports.sh [hunk 2]
```diff
@@ -113,6 +114,19 @@ function uploadFile() {
     echo "${urlKey}" #return
 }
 
+function checkThatAllTestsPassed() {
+#    content=$(wget "$url/widgets/summary.json" -q -O -)  #web request
```
Comment (AkioFaris) [inline reports.sh:118]: Do we need this comment?

**Comment by mf-code** [comment]
[5436] - DW4ZF6u3

**Comment by mf-code** [comment]
[5436] - BomI2479

**Comment by mf-code** [comment]
[5436] - Allure report on Netlify: https://5ecf5c5ff94464bae8f65f73--musing-swartz-6fc060.netlify.app

File: reports.sh (hunk 2 above)
Comment (ViktarR) [reply reports.sh]: Not really. Comment has been deleted.

File: .travis.yml (hunk 1 above)
Comment (ViktarR) [reply .travis.yml]: ok. But in that case, travis CI will break down at the last stage.

**Comment by mf-code** [comment]
[5437] - yGcvKfOk

**Comment by mf-code** [comment]
[5437] - nw0XHxlo

**Comment by mf-code** [comment]
[5437] - uK2cbPrY

**Comment by mf-code** [comment]
[5437] - o9dE0ksB

**Comment by mf-code** [comment]
[5437] - VMK0kOQA

**Comment by mf-code** [comment]
[5437] - XkdCGZwv

**Comment by mf-code** [comment]
[5437] - 4fPttYH8

**Comment by mf-code** [comment]
[5437] - Allure report on Netlify: https://5ecf6521a6566f6ed23327bc--musing-swartz-6fc060.netlify.app

**Comment by AkioFaris** [comment]
The task is done, do not merge it untill all tests are fixed. After that merge the fresh angular branch into this branch, wait for the green Travis CI and complete the PR.

**Pull request closed without merging**